pkgs/development/compilers/swift/compiler/default.nix

```diff
@@ -158,7 +149,9 @@ let
     # NOTE: @prog@ needs to be filled elsewhere.
   };
   swiftWrapper = runCommand "swift-wrapper.sh" wrapperParams ''
-    substituteAll '${../wrapper/wrapper.sh}' "$out"
+    # Make empty to avoid adding the SDK’s modules in the bootstrap wrapper. Otherwise, the SDK conflicts with the
+    # shims the wrapper tries to build.
+    darwinMinVersion="" substituteAll '${../wrapper/wrapper.sh}' "$out"
```

Comment:
The wrapper uses `darwinMinVersion` to detect whether it is targeting Linux or Darwin. On Darwin, it will add the SDK to `NIX_CFLAGS_COMPILE`. Setting it to empty triggers the Linux behavior (no setting), which is what we want because the full SDK breaks the Swift bootstrap.